paasta_tools/utils.py

```diff
@@ -633,6 +633,13 @@ def get_fsm_cluster_map(self):
             raise PaastaNotConfiguredError(
                 'Could not find fsm_cluster_map in configuration directory: %s' % self.directory)
 
+    def get_dashboard_links(self):
+        try:
+            return self['dashboard_links']
```

Comment:
Don't crash so hard on this. It is ok to just return no links if we don't have any set yet.